op-node/rollup/derive/l1_block_info.go

```diff
@@ -200,15 +215,29 @@ func (info *L1BlockInfo) marshalBinaryEcotone() ([]byte, error) {
 	return w.Bytes(), nil
 }
 
+func (info *L1BlockInfo) unmarshalBinaryIsthmus(data []byte) error {
+	return info.unmarshalBinaryEcotoneAndIsthmus(data, true)
+}
 func (info *L1BlockInfo) unmarshalBinaryEcotone(data []byte) error {
+	return info.unmarshalBinaryEcotoneAndIsthmus(data)
+}
+
+// optional isIsthmus flag
+func (info *L1BlockInfo) unmarshalBinaryEcotoneAndIsthmus(data []byte, isIsthmus ...bool) error {
```

Comment:
I think that we want to keep the standard of having 1 marshal and 1 unmarshal function per hardfork

Reply:
I have them like this:
```go
func (info *L1BlockInfo) unmarshalBinaryIsthmus(data []byte) error {
	return info.unmarshalBinaryEcotoneAndIsthmus(data, true)
}
func (info *L1BlockInfo) unmarshalBinaryEcotone(data []byte) error {
	return info.unmarshalBinaryEcotoneAndIsthmus(data)
}

// optional isIsthmus flag
func (info *L1BlockInfo) unmarshalBinaryEcotoneAndIsthmus(data []byte, isIsthmus ...bool) error {
...
```
Do you think this works? or should I have `unmarshalBinaryEcotoneAndIsthmus` not extend `L1BlockInfo`?